core/store/migrate/migrations/0188_create_automation_upkeep_state.sql

```diff
@@ -0,0 +1,18 @@
+-- +goose Up
+
+CREATE TABLE evm_upkeep_state (
+    id BIGSERIAL PRIMARY KEY,
+    evm_chain_id NUMERIC NOT NULL,
+    work_id TEXT NOT NULL,
```

Comment:
use `work_id` as the primary key

Reply:
Is that guaranteed to be unique across chains?